Dynatrace/dynatrace-operator
# Update minimum supported TLS version of webhook to TLS 1.3

# Description

Currently we support TLS version down to TLS. 
As TLS 1.3 is supported in kubernetes 1.19+ and openshift 1.6+, the PR updates the min TLS version.

## How can this be tested?

Run the operator with application monitoring enabled. - Check if the init container and DynaKube is still generated.

## Checklist
